components/Filters/editor/LabelActions.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { c } from 'ttag';
+import { Icon, SmallButton, Autocomplete, useAutocomplete } from 'react-components';
+import { noop } from 'proton-shared/lib/helpers/function';
+
+function LabelActions({ selection, labels, onChange }) {
+    const itemMapper = (label) => ({
+        label: label.Name,
+        value: label
+    });
+
+    const { changeInputValue, selectedItems, inputValue, select, deselect } = useAutocomplete({
+        multiple: true,
+        initialSelectedItems: selection,
+        onChange(selection) {
+            onChange(selection.map(({ Name }) => Name));
+        }
+    });
+
+    return (
+        <>
+            <input />
+            <Autocomplete
+                placeholder={c('Placeholder').t`Add a label ...`}
+                className="w100"
```

Comment:
The `Autocomplete` input should already have the `w100` class. If it's not the case, you can add it please?